tests/blotter/test_blotter.py

```diff
@@ -17,7 +17,7 @@
 import pandas as pd
```

Comment:
@richafrank Thank you for pointing that out! I originally had it this way because I also had a blotter/test_core.py for testing registration, but now it's no longer needed due to the RegistrationManager class.